docs/operator/getting-started.md

```diff
@@ -0,0 +1,274 @@
+# Getting Started
+
+An official Helm chart is planned to make it really easy to deploy the Grafana Agent
+Operator on Kubernetes. For now, things must be done a little manually.
+
+## Deploying CustomResourceDefinitions
```

Comment:
I'm not very familiar with the operator pattern and I was a bit lost with this part at first. I think mentioning that we're creating resources to get the operator running would clarify it.